### PR TITLE
Merak-topo Unit Test

### PR DESCRIPTION
**Pull request by zzengcs**
Add unit test for functions in merak-topo service

### CODE REVIEW

**Comment by xieus** [comment]
@zzengcs Can you describe which scenario the UT covers in the PR's description part?

File: services/merak-topo/database/database_test.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	vhost_test.OperationType = "CREATE"
+	vhost_test.Status = "DEPLOYING"
+
+	err := SetValue("topo_test_vhost", vhost_test)
+	if err != nil {
+		fmt.Println("SetValue for ComputeNode passed")
+	} else {
+		fmt.Println("SetValue for ComputeNode failed")
+	}
+}
```
Comment (xieus) [inline services/merak-topo/database/database_test.go:46]: Does the Database UT actually write to a DB?

File: services/merak-topo/grpc/service/service_test.go [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+	}
+
+	client := pb.NewMerakTopologyServiceClient(conn)
+
+	// Run Test for INFO
+	resp1, err1 := client.TopologyHandler(ctx, &topology_info)
+	if err1 != nil {
+		t.Fatalf("Topology Handler failed: %v", err1)
+	}
+	log.Printf("TopologyHandler Response: %+v", resp1)
```
Comment (xieus) [inline services/merak-topo/grpc/service/service_test.go:132]: Does this test involve DB, or purely logics test?

File: services/merak-topo/database/database_test.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+*/
+
+package database
+
+import (
+	"fmt"
+	"testing"
+
+	entities "github.com/futurewei-cloud/merak/services/merak-topo/entities"
+)
```
Comment (xieus) [inline services/merak-topo/database/database_test.go:21]: Consider to add a backward and/or forward compatibility test.

**Review by xieus — COMMENTED** [review]
@zzengcs Some comments left. Pls take a look.

**Comment by xieus** [comment]
Any update here? @zzengcs

**Comment by zzengcs** [comment]
This PR is solved by PR #187

**Pull request closed without merging**